### PR TITLE
Removes force insertion of '$ErrorActionPreference = "Continue"' if "Stop On Errors" is unchecked

### PR DESCRIPTION
**Pull request by jeremypope**
Similar to the intent of #18 to fix the behavior shown in https://issues.jenkins.io/browse/JENKINS-60423, undo the breaking change introduced in version 1.4 where it will force insert the line `$ErrorActionPreference = "Continue"` if the "Stop On Errors" checkbox is enabled, as this breaks the usage of PowerShell scriptblocks that have advanced parameter blocks at the beginning that contain `[CmdletBinding()]` and/or `param()` blocks since these must be the first executable line within a scriptblock if given. 

However, this differs from #18 where instead of moving to a choice between `Stop`, `Continue`, or no insertion at all, this branch instead just removes the insertion if the option is unchecked, as the default behavior of PowerShell's `$ErrorActionPreference` is `Continue`.

tl;dr - Keep the plugin interface and intended behavior the same, but remove the breaking behavior of forcing `$ErrorActionPreference = "Continue"` into the beginning of the script since this is a default setting

### CODE REVIEW

**Comment by jeremypope** [comment]
@froque Is there any additional information I need to provide for this?
